saleor/discount/migrations/tasks/saleor3_16.py

```diff
@@ -1,9 +1,14 @@
 from dataclasses import dataclass
 from typing import Dict, List
 from ....celeryconf import app
```

Comment:
https://docs.saleor.io/docs/3.x/developer/community/zero-downtime-migrations#additional-autodiscovery-with-specified-related-names

additional configuration is needed in this case

Reply:
It's already added.